x-pack/filebeat/module/fortinet/clientendpoint/test/generated.log-expected.json

```diff
@@ -0,0 +1,5702 @@
+[
+    {
+        "@timestamp": "2020-01-29T08:09:59.000Z",
+        "destination.ip": [
+            "10.102.123.34"
+        ],
+        "destination.port": 3994,
+        "event.action": "deny",
+        "event.code": "http",
+        "event.dataset": "fortinet.clientendpoint",
+        "event.module": "fortinet",
+        "event.original": "January 29 06:09:59 boNemoe4402.www.invalid proto=udp service=http status=deny src=10.150.92.220 dst=10.102.123.34 src_port=7178 dst_port=3994 server_app=reeufugi pid=7880 app_name=enderitq traff_direct=sperna block_count=884 logon_user=billoi@oreetdol1714.internal.corp msg=failure",
+        "event.outcome": "Failure",
```

Comment:
This should be lowercased, also what does `Failure` mean here? That a user/process was prevented from issuing an HTTP request?

Reply:
I've mapped event.outcome to the expected values.